src/services/rns/rns.processor.ts

```diff
@@ -3,7 +3,7 @@ import config from 'config'
 import { Eth } from 'web3-eth'
 import { EventData } from 'web3-eth-contract'
 import Utils from 'web3-utils'
-import { RnsServices } from '.'
+import { RnsService, RnsServices } from '.'
```

Comment:
Double import :)

Reply:
no.. one is in plural the other in singular. Because now im passing an individual service as parameter and need the type

Reply:
Yeah, it is bit a bad naming. Maybe we should rename it to `RnsBaseService`? As the `RnsServices` are derived from that.

Reply:
Yes will do that.

Reply:
Renamed it.